### PR TITLE
Fixed Triple click selection and copy&paste in read-only mode

### PR DESCRIPTION
**Pull request by sennpang**
**Description**
Triple click selection and copy&paste in read-only mode

**Issue**
Fixes: #4897 #3871

**Example**
before:
https://user-images.githubusercontent.com/66263028/158982608-bada507e-2249-4b70-9c6c-906f687ee36a.mov
after:
https://user-images.githubusercontent.com/16861647/160055330-4df1837d-c364-4bc8-96e0-f4a5297c740e.mov

**Context**

**Checks**
- [x] The new code matches the existing patterns and styles.
- [x] The tests pass with `yarn test`.
- [x] The linter passes with `yarn lint`. (Fix errors with `yarn fix`.)
- [x] The relevant examples still work. (Run examples with `yarn start`.)
- [ ] You've [added a changeset](https://github.com/atlassian/changesets/blob/master/docs/adding-a-changeset.md) if changing functionality. (Add one with `yarn changeset add`.)

### CODE REVIEW

File: packages/slate-react/src/components/editable.tsx [hunk 1]
```diff
@@ -733,14 +733,25 @@ export const Editable = (props: EditableProps) => {
...
               if (
-                !readOnly &&
                 hasTarget(editor, event.target) &&
                 !isEventHandled(event, attributes.onClick) &&
                 isDOMNode(event.target)
               ) {
                 const node = ReactEditor.toSlateNode(editor, event.target)
                 const path = ReactEditor.findPath(editor, node)
 
+                if (event.detail === 3 && readOnly) {
```
Comment (dylans) [inline packages/slate-react/src/components/editable.tsx:743]: In general, I prefer an enum so I don't have to remember what 3 is... even if it's just the line before, e.g. `const TRIPLE_CLICK = 3` and then `if (event.detail === TRIPLE_CLICK ...`

**Review by dylans — CHANGES_REQUESTED** [review]
See my suggestion. Also please add a changeset. Thanks!

File: packages/slate-react/src/components/editable.tsx (hunk 1 above)
Comment (sennpang) [reply packages/slate-react/src/components/editable.tsx]: ok

File: .changeset/khaki-candles-serve.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"slate-react": patch
```
Comment (dylans) [inline .changeset/khaki-candles-serve.md:2]:
```suggestion
'slate-react': patch
```

File: .changeset/khaki-candles-serve.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+"slate": patch
```
Comment (dylans) [inline .changeset/khaki-candles-serve.md:3]:
```suggestion
'slate': patch
```

**Review by dylans — CHANGES_REQUESTED** [review]

**Review by dylans — APPROVED** [review]

**Pull request merged**